tests/e2e/config/bootstrap.js

```diff
@@ -122,6 +122,40 @@ function observeConsoleLogging() {
 	} );
 }
 
+/**
+ * Runs Axe tests when the block editor is found on the current page.
+ *
+ * @return {?Promise} Promise resolving once Axe texts are finished.
+ */
+async function runAxeTestsForBlockEditor() {
+	if ( ! await page.$( '.block-editor' ) ) {
```

Comment:
Already looking forward to apply these tests to other parts of the AMP plugin as well 💥